### ISSUE TITLE
Port ElectricalAge source into MCreator [Very Long Term]

### ISSUE THREAD

**Issue by ellingtonvengaboysmuseum** [jrddunbr/ElectricalAge#181]
This is a very long term project... Port everything from and the ElectricalAge source coding and mod into MCreator

MCreator is a highly advanced and sophisticated minecraft mod making tool that is in development that works very well for coding and programming illiterates similar to adobe dream weaver for website creation

if this was done, Mcreator could exponentially speed up development of future versions of electrical age since you could get a much larger development team from people like me who don't know much about coding using Mcreator.

**In a few days i may post a video on here showing off just how advanced mcreator is and some of the interesting things mcreator can do with sounds and custom dimensions and teleporting the player to them without a portal that would take a lot longer to pull off writing the code in entirely java...**

**Comment by jrddunbr** [comment]
While that sounds like a good idea, you have (as far as I can tell) 0 understanding of the codebase.

Tools like that are not going to solve our problems. Tools like that are great for basic mods, not ones that are incredibly complex like ours. There would not be a way to translate the code from this mod into a tool like that. The other major issue is programs like that are incredibly inefficient compared to a real set of programmers that are aware of optimal computer algs and shortcuts to make our code run fast as possible.

TL;DR, you're trying to make a difficult program from a cookie cutter solution. I bet our code has more lines than that entire program.

If you think I'm wrong, I dare you to do it.

**Comment by jrddunbr** [comment]
Also, mods like Electrical Age require good developers to keep things running smoothly.

I doubt most people who want to be developers (that you suggest would suddenly be able to do it because it uses some tool to make it point and click) understand what a current source is, and a (mostly) complete understanding of electrical components and modified nodal analysis is required for an effective mod.

For example, I doubt the new "developers" could write code (even in the program you suggest, if it were even possible) to do something such as converting a Thevenin system to a Norton's system for an optimal electrical solution (in certain scenarios this is optimal).
